src/menu.c

```diff
@@ -435,14 +443,16 @@ int gophermap(state *st, char *mapfile, int depth)
 		}
 	}
 
+CLOSE_FP:
 	/* Clean up & return */
+
 #ifdef HAVE_POPEN
 	if (exe & st->opt_exec) pclose(fp);
 	else
 #endif
 		fclose(fp);
```

Comment:
While we are here we can probably remove the extra tab here :), if you want to include that.

Reply:
Done